config/rbac/role.yaml

```diff
@@ -24,18 +24,6 @@ rules:
   - patch
   - update
   - watch
-- apiGroups:
-  - datamover.oadp.openshift.io
-  resources:
-  - '*'
-  verbs:
-  - '*'
-- apiGroups:
-  - volsync.backube
-  resources:
-  - '*'
-  verbs:
-  - '*'
 - apiGroups:
   - snapshot.storage.k8s.io
```

Comment:
We might not need the storage API group permissions as well.

Reply:
will investigate

Reply:
I was not able to test yet, but I was wondering if that is still needed for CSI path

Reply:
I think you are right @shubham-pampattiwar 

I ran the 3 CSI and 2 DataMover E2E tests removing lines 27 through 62 of this file (removing `snapshot.storage.k8s.io` and `storage.k8s.io` rules) and the tests passed (this is enough to confirm that they are not necessary, right?)

My question: is why we do not need it anymore?

Reply:
We needed them for VSM controller, we created clones of VS and VSC during the VSM datamover backup process.

Reply:
alright, will remove in following commit
Thanks!